### PR TITLE
Add abstract and motivation

### PR DESCRIPTION
**Pull request by ThunderDeliverer**
This PR adds the following things:

- [x] Extends the name
- [x] Adds description
- [x] Adds abstract
- [x] Adds Motivation
- [x] Emphasises keywords in Specification

### CODE REVIEW

**Comment by ThunderDeliverer** [comment]
While this is still a work in progress, we can start the naming discussion while the last two parts of the PR are being added.

File: README.md [hunk 1]
```diff
@@ -13,6 +13,9 @@ requires: 165, 721
 
 ## Abstract
 
+The Parent-Governed Nestable NFT standard allows for a new inter-NFT relationship and interaction.
+
+An NFT can be owned by a single other NFT, but can in turn have a number of NFTs that it owns. This proposal established the framework for the parent-child relationships of NFTs. A parent token is the one that owns another token. A child token is the token that is owned by another token. A token can be both a parent and child at the same time. Child tokens of a given tokens can be fully managed by the parent token's owner, but can be proposed by anyone.
```
Comment (steven2308) [inline README.md:18]: Maybe `establishes` instead of  `established`

Comment (steven2308) [reply README.md:18]: I'd like to make it obvious from here that sending a token to another implies giving up ownership. But with your words :sweat_smile:

Comment (ThunderDeliverer) [reply README.md:18]: I agree this sounds better 👌

**Review by steven2308 — APPROVED** [review]
Looking great! Can we merge this now?

**Pull request merged**